### ISSUE TITLE
Kitti Viewer issue

### ISSUE THREAD

**Issue by hefc2016** [traveller59/second.pytorch#385]
Hello

Now I have trained the Kitti dataset follow the read me and got the model file. And I use Web KItti Viewer to visualize the result. I can open Web KItti Viewer and see the image/point cloud/object, but have some problems compared to demo image in readme.

1. can't render 360 degree data in bird's eye view and point cloud, only part of data, seems 75 degree FOV, I change the FOV parameter in Web KItti Viewer but it does not change amnything
2. no bounding box display in Image window

Could anyone help me for this issues?
![Screenshot from 2020-06-25 17-47-11](https://user-images.githubusercontent.com/24312605/85698470-464b0180-b70d-11ea-8e9c-d6aa563a29df.png)

**Comment by AngYikHong** [comment]
Hi, do you solve the problems? I have the same problem with you.

**Comment by hefc2016** [comment]
The bounding box in image has been removed in V1.6, you change to use V1.5 or earlier branch, then you can see 360 degree FOV lidar data and bbox in image. That's what I have tried from my side.

**Comment by AngYikHong** [comment]
Hi, thanks for your information. May I know how can I change the branch to V1.5 and do I have to retrain everything again? Sorry I am still a newbie.

**Comment by hefc2016** [comment]
you can use below command to clone v1.5 repo:
`git clone -b v1.5 https://github.com/traveller59/second.pytorch.git`

I think you don't need to retrain the model, you can just input the data and model which create using v1.6.

**Comment by AngYikHong** [comment]
Hi @hefc2016 , thanks for your information. May I know how you evaluate your model on test data? I change the eval_input_reader 'kitti_info_path' with kitti_infos_test.pkl in the all.fhd.config file ,then run 'python ./pytorch/train.py evaluate...'，it can't show any evaluation results. Also, when I put kitti_infos_test.pkl as my infoPath and key in the image index but there are not any ground truth box on the point cloud and BEV. Sorry for disturbance.

**Comment by arxidinakbar** [comment]
@AngYikHong  I believe the reason you cannot get evaluation results because of there are no  label files( the ground truth) for test set. you could only get final evaluation by submit your result on KITTI webside, and  they would do this for you online.
